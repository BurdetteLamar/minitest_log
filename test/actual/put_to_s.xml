<log>
  <data_ name='put_to_s' class='Float' method=':to_s'>
    3.14159
  </data_>
</log>
