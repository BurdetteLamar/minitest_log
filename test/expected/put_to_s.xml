<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <data name='to_s' class='Float' method=':to_s'>
    3.14159
  </data>
</log>
